### ISSUE TITLE
Specify multiple areas?

### ISSUE THREAD

**Issue by StephenPrice41** [tabulapdf/tabula-java#227]
Looking at Usage Examples at https://github.com/tabulapdf/tabula-java I see;
_To specify multiple areas, -a option should be repeated. Default is entire page_
So I was trying to select multiple areas with no success. It would only process the first area.
After much hair pulling I ran -h locally and I only see;
_-a,--area <AREA>           Portion of the page to analyze (top,left,bottom,right). Example: --area 269.875,12.75,790.5,561. Default is entire page_
As far as I can tell I am running the same version;
java -jar tabula-1.0.1-jar-with-dependencies.jar -v
_tabula 1.0.1 (c) 2012-2017 Manuel Aristarán_
I am running it in Windows 7.
Any ideas?

**Comment by StephenPrice41** [comment]
There has been a new release which does seem to fix this issue.
https://github.com/tabulapdf/tabula-java/releases/tag/v1.0.2
I was visiting this page from time to time (even though I did subscribe) hoping for a fix of some sort, and it wasn't until I visited the homepage today did I see that there was a new release.
The new version does allow me to select an area but spits out the following before the results (-i or --silent does not remove the output).

May 30, 2018 3:38:16 PM org.apache.pdfbox.pdmodel.graphics.color.PDDeviceRGB suggestKCMS
INFO: To get higher rendering speed on JDK8 or later,
May 30, 2018 3:38:16 PM org.apache.pdfbox.pdmodel.graphics.color.PDDeviceRGB suggestKCMS
INFO:   use the option -Dsun.java2d.cmm=sun.java2d.cmm.kcms.KcmsServiceProvider
May 30, 2018 3:38:16 PM org.apache.pdfbox.pdmodel.graphics.color.PDDeviceRGB suggestKCMS
INFO:   or call System.setProperty("sun.java2d.cmm", "sun.java2d.cmm.kcms.KcmsServiceProvider")
